tests/llmcompressor/modeling/test_calib_glm4_moe_lite.py

```
@@ -0,0 +1,115 @@
+    config = _tiny_config()
+    with torch.device("cuda"):
+        original = Glm4MoeLiteMoE(config)
+        for param in original.parameters():
+            param.data.normal_(mean=0.0, std=0.02)
+
+    module = CalibrationGlm4MoeLiteMoE(original, config, calibrate_all_experts=True)
+
+    num_experts = len(module.experts)
+    expert_triggered = [False for _ in range(num_experts)]
+
+    def hook_fn(i, module, input, output):
```

Ruff (0.15.9)

[error] Function argument `input` is shadowing a Python builtin

(A002)